### PR TITLE
Doorlogic doublecheck

### PR DESCRIPTION
**Pull request by AlmostSeagull**
Hint door logic fixes 
Made the jump into the tns alcove in Japes in logic unconditionally

### CODE REVIEW

File: randomizer/Lists/DoorLocations.py [hunk 1]
```diff
@@ -739,7 +735,7 @@ def GetBossLobbyRegionIdForRegion(region_id, region):
             kong_lst=[Kongs.donkey],
             group=2,
             moveless=False,
-            logic=lambda l: (l.isdonkey and l.strongKong) or l.phasewalk or l.generalclips,
+            logic=lambda l: l.isdonkey and l.strongKong,
```
Comment (AlmostSeagull) [inline randomizer/Lists/DoorLocations.py:738]: This door location is halfway the quicksand tunnel. Glitches won't save you from getting your feet hot. (as far as I have tried, you cannot talk to wrinkly from within the wall)

File: randomizer/Lists/DoorLocations.py [hunk 2]
```diff
@@ -750,7 +746,7 @@ def GetBossLobbyRegionIdForRegion(region_id, region):
...
```
Comment (AlmostSeagull) [inline randomizer/Lists/DoorLocations.py:749]: This door location is not in the quicksand tunnel. Glitches won't save you from getting your feet hot. (as far as I have tried, you cannot talk to wrinkly from within the wall. Mind, this is the wall of the podium with stairs near Snides with the tag barrel above it)

File: randomizer/Logic.py [hunk 3]
```diff
@@ -647,7 +647,7 @@ def HintAccess(self, location, region_id):
             return False
         # The only weird exception: vanilla Fungi Lobby hint doors only check for Chunky, not the current Kong, and all besides Chunky's needs grab
         if not self.settings.wrinkly_location_rando and not self.settings.remove_wrinkly_puzzles and region_id == RegionEnum.FungiForestLobby:
-            return self.chunky and (location.kong == Kongs.chunky or (self.donkey and self.grab))
+            return (self.chunky or self.settings.wrinkly_available) and (location.kong == Kongs.chunky or (self.donkey and self.grab))
```
Comment (pbeierle22) [inline randomizer/Logic.py:650]: The original is very weird but correct. All you need is Chunky (plus DK and grab) to be able to read all the hint doors in vanilla Fungi hint doors.

Comment (AlmostSeagull) [reply randomizer/Logic.py:650]: Yes, but if you have kongless hint doors enabled you don't need Chunky to be unlocked in order to read the hints. Any kong (and if not the chunky door, DK and grab) will do with kongless hint doors. That's what I'm getting at in this change.

Comment (AlmostSeagull) [reply randomizer/Logic.py]: Appearantly kongless hint doors doesn't affect Chunky's door on the wheel. Going to err on the side of caution and revert this change.

**Review by theballaam96 — APPROVED** [review]

**Pull request merged**